content/docs/dvclive/api-reference/log.md

```diff
@@ -0,0 +1,55 @@
+# dvclive.log()
+
+Generates _metrics logs_ (usable by `dvc plots`) by saving the given `name`:
+`val` pair to a `.tsv` file.
+
+```py
+ def log(name: str, val: float, step: int = None):
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+dvclive.log("loss", 0.9)
+```
+
+## Description
+
+The first call to `dvclive.log(name, val)` will create a new file in
+`{path}/{name}.tsv` including the header and first row.
+
+For example `dvclive.log("loss", 0.9)` will create `{path}/loss.tsv`:
+
+```
+timestamp step  loss
+1623671484747 0 0.9
+```
+
+Each subsequent call to `dvclive.log(name, val)` will add a new row to
+`{path}/{name}.tsv`.
+
+The created file `{path}/{name}.tsv` is usable by `dvc plots`.
+
+💡 If `name` contains slashes (i.e. `train/loss`), the required subfolders will
+be created and the file will be saved inside the last subfolder (i.e.
+`{path}/train/loss.tsv`).
+
+💡 If you call `dvclive.log` without calling
+[`dvclive.init`](/doc/dvclive/api-reference/init) first, `dvclive` will
+automatically initialize itself using either default values or environment
+variables (when used alongside `DVC`).
```

Comment:
Again, should probably mention `init` (and this info) at the beginning of the **Description**. And no need to explain DVC environment here as that should be part of the `init()` ref.

Reply:
Also, we currently use 💡 in a slightly [different manner](https://dvc.org/doc/user-guide/contributing/docs#general-language-guidelines). Up to you but if you want to be consistent with DVC docs for now I'd avoid the emoji here. Could be a note though (md block quote).